cover.sh

```diff
@@ -1,36 +1,3 @@
 #!/usr/bin/env bash
-
-function die() {
-  echo $*
-  exit 1
-}
-
-# Initialize coverage.out
-echo "mode: count" > coverage.out
-
-# Initialize error tracking
-ERROR=""
-
-declare -a packages=('' \
-    'cmd/gostatsd' 'cmd/tester'
-    'pkg/backends' 'pkg/backends/sender'
-    'pkg/backends/datadog' 'pkg/backends/graphite' 'pkg/backends/null' \
-    'pkg/backends/statsdaemon' 'pkg/backends/stdout' 'pkg/backends/newrelic'  \
-    'pkg/backends/influxdb' \
-    'pkg/cloudproviders' 'pkg/cloudproviders/aws' 'pkg/cloudproviders/k8s' \
-    'pkg/fakesocket' 'pkg/statsd' \
-    'pkg/util' 'pkg/stats' 'pkg/cluster/nodes' 'pkg/web');
-
-# Test each package and append coverage profile info to coverage.out
-for pkg in "${packages[@]}"
-do
-    go test -v -covermode=count -coverprofile=coverage_tmp.out "github.com/atlassian/gostatsd/$pkg" || ERROR="Error testing $pkg"
-    tail -n +2 coverage_tmp.out >> coverage.out 2> /dev/null ||:
-done
-
-rm -f coverage_tmp.out
-
-if [ ! -z "$ERROR" ]
-then
-    die "Encountered error, last error was: $ERROR"
-fi
+set -euo pipefail
```

Comment:
Current master build is failing because when I added InfluxDB, I added missing packages to the cover.sh.  Those packages were then removed in the subsequent refactoring, which means the coverage fails to run.

While unplanned, this new cover.sh will fix master when it's merged.